app/src/main/java/org/dslul/openboard/inputmethod/latin/userdictionary/UserDictionaryAddWordFragment.java

```diff
@@ -81,14 +88,30 @@ public View onCreateView(final LayoutInflater inflater, final ViewGroup containe
 
     @Override
     public void onCreateOptionsMenu(final Menu menu, final MenuInflater inflater) {
+        final Drawable deleteIcon = getBitmapFromVectorDrawable(R.drawable.ic_delete, 24);
+        final MenuItem actionItemDelete = menu.add(0, OPTIONS_MENU_DELETE, 0,
+                R.string.user_dict_settings_delete).setIcon(deleteIcon);
+        actionItemDelete.setShowAsAction(
+                MenuItem.SHOW_AS_ACTION_IF_ROOM | MenuItem.SHOW_AS_ACTION_WITH_TEXT);
+        if (actionItemDelete.getIcon() != null)
+            actionItemDelete.getIcon().setColorFilter(getResources().getColor(R.color.foreground_weak), PorterDuff.Mode.SRC_ATOP);
+
         final MenuItem actionItemAdd = menu.add(0, OPTIONS_MENU_ADD, 0,
                 R.string.user_dict_settings_add_menu_title).setIcon(R.drawable.ic_plus);
         actionItemAdd.setShowAsAction(
                 MenuItem.SHOW_AS_ACTION_IF_ROOM | MenuItem.SHOW_AS_ACTION_WITH_TEXT);
-        final MenuItem actionItemDelete = menu.add(0, OPTIONS_MENU_DELETE, 0,
-                R.string.user_dict_settings_delete).setIcon(android.R.drawable.ic_menu_delete);
-        actionItemDelete.setShowAsAction(
-                MenuItem.SHOW_AS_ACTION_IF_ROOM | MenuItem.SHOW_AS_ACTION_WITH_TEXT);
+    }
+
+    // The bin icon is too big compared to the plus icon; we need to reduce it.
+    // We therefore need to convert the Vector drawable image to Bitmap.
+    private BitmapDrawable getBitmapFromVectorDrawable(int drawable, float size) {
+        final int px = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, size + 0.5f, getResources().getDisplayMetrics());;
+        Drawable vectorDrawable = ContextCompat.getDrawable(getContext(), drawable);
+        vectorDrawable.setBounds(0, 0, /*right*/ px, /*bottom*/ px);
```

Comment:
Setting drawable size in pixels may look awkward, depending density of the device.

Reply:
With this it will be better (tested on my phone):

```java
private BitmapDrawable getBitmapFromVectorDrawable(int drawable, float scale) {
    Drawable vectorDrawable = ContextCompat.getDrawable(getContext(), drawable);
    final int h = (int) (scale * vectorDrawable.getIntrinsicHeight());
    final int w = (int) (scale * vectorDrawable.getIntrinsicWidth());
    vectorDrawable.setBounds(0, 0, /*right*/ w, /*bottom*/ h);
    Bitmap bitmap = Bitmap.createBitmap(/*width*/ w, /*height*/ h, Bitmap.Config.ARGB_8888);
    Canvas canvas = new Canvas(bitmap);
    vectorDrawable.draw(canvas);
    return new BitmapDrawable(getResources(), bitmap);
}
```

Reply:
Done with commit da2d29f.